Source/Initialization/WarpXInitData.cpp

```diff
@@ -111,6 +111,232 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
+      amrex::Print() << "                      |   - current_centering_nox = " << WarpX::current_centering_nox << "\n";
+      amrex::Print() << "                      |   - current_centering_noy = " << WarpX::current_centering_noy << "\n";
+      amrex::Print() << "                      |   - current_centering_noz = " << WarpX::current_centering_noz << "\n";
+    }
+    else if (dims=="2"){
+      amrex::Print() << "                      |   - field_centering_nox = " << WarpX::field_centering_nox << "\n";
+      amrex::Print() << "                      |   - field_centering_noz = " << WarpX::field_centering_noz << "\n";
+      amrex::Print() << "                      |   - current_centering_nox = " << WarpX::current_centering_nox << "\n";
+      amrex::Print() << "                      |   - current_centering_noz = " << WarpX::current_centering_noz << "\n";
+     }
+    else if (dims=="1"){
+      amrex::Print() << "                      |   - field_centering_noz = " << WarpX::field_centering_noz << "\n";
+      amrex::Print() << "                      |   - current_centering_noz = " << WarpX::current_centering_noz << "\n";
+     }
+    }
+    if (WarpX::use_hybrid_QED == true){
+      amrex::Print() << "                      | - use_hybrid_QED = true \n";
+    }
+
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD){
+    // Print solver's order
+      std::string psatd_nox_fft, psatd_noy_fft, psatd_noz_fft;
+      psatd_nox_fft = (nox_fft == -1) ? "inf" : std::to_string(nox_fft);
+      psatd_noy_fft = (noy_fft == -1) ? "inf" : std::to_string(noy_fft);
+      psatd_noz_fft = (noz_fft == -1) ? "inf" : std::to_string(noz_fft);
+
+      if (dims=="3" ){
+        amrex::Print() << "Spectral order:       | - psatd.nox = " << psatd_nox_fft << "\n";
+        amrex::Print() << "                      | - psatd.noy = " << psatd_noy_fft << "\n";
+        amrex::Print() << "                      | - psatd.noz = " << psatd_noz_fft << "\n";
+      }
+      else if (dims=="2" and WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD){
+        amrex::Print() << "Spectral order:       | - psatd.nox = " << psatd_nox_fft << "\n";
+        amrex::Print() << "                      | - psatd.noz = " << psatd_noz_fft << "\n";
+      }
+      else if (dims=="1" and WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD){
+        amrex::Print() << "Spectral order:       | - psatd.noz = " << psatd_noz_fft << "\n";
+      }
+    }
+    else{
+      amrex::Print() << "    ***********************************************************************\n";
+      amrex::Print() << "    *    In order to use Yee or CKC or ECT solver, WarpX needs to be      *\n" <<
+      "    *     compiled with a finite-difference (FDTD) Maxwell solver         *\n";
+      amrex::Print() << "    ***********************************************************************\n";
+    }
```

Comment:
This `else` branch I do not understand: WarpX is always compiled with FDTD solvers, only spectral solvers and EB are opt-in.